pq-crypto/EVP_pq_constants.h

```diff
@@ -0,0 +1,38 @@
+//
+// Created by lamjyoti on 8/11/2021.
+//
+
+#ifndef AWSLC_EVP_PQ_CONSTANTS_H
+#define AWSLC_EVP_PQ_CONSTANTS_H
+
+
+/* TLDR s2n: 0=true, -1=false,
+ * TLDR aws-lc: 1=true, 0=false
+ * current work around to keep following macros */
+/* A value which indicates the outcome of a function */
+#define EVP_PQ_SUCCESS 1
+#define EVP_PQ_FAILURE 0
+
+#define IN /* Indicates a necessary function input */
+#define OUT /* Indicates a function output */
```

Comment:
Are we using these macros anywhere in this code? (I didn't see them, but maybe I missed them.)